### PR TITLE
OKTA-221199: Add Java Springboot Code Sample for Signout

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
## Description:
- **What's changed?** 
Added Java code snippet sample for User Signout scenario.

- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-221199](https://oktainc.atlassian.net/browse/OKTA-221199)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+
+<a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
+
+1. Select **Applications**, and then select your application.
+
+2. Select **General** and click **Edit**.
+
+3. In the **Logout redirect URI section**, add the **Base URI** of your application. You can optionally follow that URI with a path, for example, `http://localhost:8080/logged_out`. Also, add any URIs where your application runs in production, such as `https://app.example.com/logged_out`.
+
+4. Click **Save**.
```
Comment (bdemers) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md:15]: IIRC, this file should only contain the spring specific bits

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+                .and().oauth2Login();
+    }
+}
+```
+
+If you are using the [Okta Spring Boot Starter](https://github.com/okta/okta-spring-boot), you can configure an RP-Initiated (SSO) Logout by setting the `okta.oauth2.postLogoutRedirectUri` property with an absolute URI such as:
+
+```properties
+okta.oauth2.postLogoutRedirectUri=http://localhost:8080/
+```
```
Comment (bdemers) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md:68]: My suggestion is to only show the properties option and add a link to the full spring doc for more details.  All of the other guides assume the use of the `okta-spring-boot-starter`

I think there might be a missing page too, there should be snippets for both of these sections (i could be missing something though):
- https://developer.okta.com/docs/guides/sign-users-out/springboot/define-signout-callback/
- https://developer.okta.com/docs/guides/sign-users-out/springboot/sign-out-of-okta/


I'm not sure what would go in the `define-signout-callback`, as this is something that Spring Security (and other frameworks) do automatically,

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md]: Addressed.

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md [hunk 3]
```diff
@@ -1 +1,7 @@
-This functionality is coming soon. Watch this GitHub issue for updates: [#126 Support OIDC logout](https://github.com/okta/okta-spring-boot/issues/126)
+You can configure an RP-Initiated (SSO) Logout by setting the `okta.oauth2.postLogoutRedirectUri` property with an absolute URI such as:
```
Comment (bdemers) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md:1]: I think our other docs mention that this should be placed in `application.properties`, double-check and make sure the wording is the same

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md [hunk 4]
```diff
@@ -1 +1,7 @@
...
+
+```properties
+okta.oauth2.postLogoutRedirectUri=http://localhost:8080/
```
Comment (bdemers) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md:4]:
```suggestion
okta.oauth2.post-logout-redirect-uri=http://localhost:8080/
```

As of the current major version of Spring, kabob case is recommended.

Comment (bdemers) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md:4]: Actually, i just tried this, and it didn't work.  I'm 99% sure the fix is trivial:
https://github.com/okta/okta-spring-boot/issues/175

I try to make time to try that later today

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md (hunk 3 above)
Comment (arvindkrishnakumar-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md]: You're right! Updated!

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md (hunk 2 above)
Comment (arvindkrishnakumar-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md]: Updated _define-signout-callback_ page with content (link to Spring Security official documentation).

_sign-out-of-okta_ content for remote andlocal logout is already updated.

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md (hunk 4 above)
Comment (arvindkrishnakumar-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/springboot/remotesignout.md:4]: Will address okta/okta-spring-boot#175 in a separate PR (outside the scope of this PR) to keep this PR unblocked.

**Pull request merged**